src/cdk/stepper/stepper.ts

```diff
@@ -28,12 +28,14 @@ import {
   ChangeDetectionStrategy,
   ChangeDetectorRef,
   OnChanges,
+  OnDestroy
 } from '@angular/core';
 import {LEFT_ARROW, RIGHT_ARROW, ENTER, SPACE} from '@angular/cdk/keycodes';
 import {CdkStepLabel} from './step-label';
 import {coerceBooleanProperty} from '@angular/cdk/coercion';
 import {AbstractControl} from '@angular/forms';
 import {Direction, Directionality} from '@angular/cdk/bidi';
+import {takeUntil} from 'rxjs/operators/takeUntil';
```

Comment:
I think this import should be in `src/lib/stepper/stepper.ts` since it isn't used in this file.

Reply:
Agreed, was a mistake.